### ISSUE TITLE
Backdoor

### ISSUE THREAD

**Issue by luhensis** [H1R0GH057/Anonymous#4]
This program is a HTTP/HTTPS backdoor into your computer! Dont download!

**Comment by jcrommar** [comment]
> This program is a HTTP/HTTPS backdoor into your computer! Dont download!

Which one? You mean the whole repo?

**Comment by luhensis** [comment]
Yeah, I think so. When I downloaded it, Windows Security said it was and it was sending info to a remote computer.

**Comment by deeshirazi** [comment]
well then it is @NetworkChuck(yYouTube) who uploaded this here to use his fans to do his dirty work1 Is there anyone here who watched Networkchuck video and clicked on the link on his video description? #NetworkChuck any explanation?

**Comment by luhensis** [comment]
No, NetworkChuck found it here for free.

**Comment by deeshirazi** [comment]
> No, NetworkChuck found it here for free.

how do you know?

**Comment by deeshirazi** [comment]
> No, NetworkChuck found it here for free.

actually i just remembered NetworkChuck said he "bought" the script on the Darkweb, he said he paid bitcoin for it remember? And you say windows security said it´s a backdoor to your pc that controlls from somewhere else and the funny thing is that NetworkChuck showed us how he had build his bots on BYOB!!! What a quensidence

**Comment by endivuer** [comment]
> This program is a HTTP/HTTPS backdoor into your computer! Dont download
i alredy run some of the scripts on my mac  F

**Comment by endivuer** [comment]
> > This program is a HTTP/HTTPS backdoor into your computer! Dont download
> > i alredy run some of the scripts on my mac  F

how do i remove

**Comment by Elara6331** [comment]
A git repository on its own is simply incapable of being a backdoor. It is very much possible, although unlikely, that one of these scripts is a backdoor, but if you infected yourself with it, it would be entirely your fault as the code is here, and you can read it. If it was all in C, I would understand, but most of it is in Perl and python, some of the easiest languages to read. The only C file is not a compiled binary, and is therefore harmless. Windows security is a horrible program as is windows itself and I would argue that even if this was a backdoor, windows is more of a backdoor than it can ever be.

**Comment by ghost** [comment]
so its safe?

**Comment by Elara6331** [comment]
I am not guaranteeing all the scripts are safe, I am simply saying downloading the repo itself is entirely safe. As always, you should either read the code or run it in a VM/Container before trusting it, but just downloading it is safe

**Comment by ghost** [comment]
is it safe to run it in a remotedesktop?

**Comment by Elara6331** [comment]
Not unless you don't care about the computer behind the Remote Desktop. Like I said, it is unlikely the scripts are malicious, but possible, you can try running it on a service like [replit](https://repl.it).

**Comment by 3x1l3l4bs** [comment]
@NetworkChuck is an script kiddie and leads an army of wanna be hackers like yourselves many of witch don't know the first thing about what it really means to be a hacker or the Philosophy behind it, most of you just wanna download the tools to look cool and win CSGO matches because you suck to much to actually win without the aid of shitily made cheats and "hacks" OR you guys just wanna brag to your friends about how 1337 you are  all of you should google (or rather duckduckgo) ethical hacking 

@Arsen6331 this excludes you you seem to be the only one here with a brain

**Comment by deeshirazi** [comment]
> @NetworkChuck is an script kiddie and leads an army of wanna be hackers like yourselves many of witch don't know the first thing about what it really means to be a hacker or the Philosophy behind it, most of you just wanna download the tools to look cool and win CSGO matches because you suck to much to actually win without the aid of shitily made cheats and "hacks" OR you guys just wanna brag to your friends about how 1337 you are all of you should google (or rather duckduckgo) ethical hacking
> 
> @Arsen6331 this excludes you you seem to be the only one here with a brain

You took it personally i don´t know why! Why should a person upload such thing to use his "Fans" as an army? And i think this repository has nothing to do with hacking, it´s DoS/DDoS scripts that alone does not mean shite! Even if you are so good at "hacking" there are always people who are better so there is nothing to brag about! Calm down!

**Comment by deeshirazi** [comment]
> so its safe?

Just reinstall your windows and ALWAYS use a virtual machine for these kind of stuff!

**Comment by jcrommar** [comment]
This started to feel like Facebook with all the offensive comments. If you disagree with something there are 1 billion different ways to voice it out and give your opinion. 

Grow up! If someone wants to look cool, let them be and have their machines get hacked. Security is not something to play but if they don't know, how is that your problem?

**Comment by 3x1l3l4bs** [comment]
"You took it personally i don´t know why! Why should a person upload such thing to use his "Fans" as an army? And i think this repository has nothing to do with hacking, it´s DoS/DDoS scripts that alone does not mean shite! Even if you are so good at "hacking" there are always people who are better so there is nothing to brag about! Calm down!".  

"You took it personally i don´t know why" 
I did not take it personally i'm just tired of shit like this popping up around the internet and skids calling themselves hackers and people believing it. and not even knowing there are people out there who use what they know for good and that hacking isn't simply downloading some one else's tools/code and using it 

"there is nothing to brag about! " 
I'm not bragging i'm just mad that people now think hacking is DDOS/DOS attacks, that its downloading something then calling yourself a hacker it's people like theses that give the hacker culture a bad rap 

"there are always people who are better"
Im not saying i'm better than anyone from the time i wrote my first line of code i've accepted this
i've never to this day called myself an expert or a pro.

**Comment by Sys-key-design** [comment]
Seeing stuff like this online isn't as bad as you are explaining it to be. Some people download the code to study it and see how it works. Many people could actually be learning from some of this. But I agree that it is genuinely irritating that people download stuff like this and claim they are "hackers", and people ending up believing them. No hard feelings.

:D

**Comment by Jinja-Ninja** [comment]
> so its safe?

Next time read the code before asking questions like this.

**Comment by Yoshida35** [comment]
For anyone who sees this comment, it is safe because you can always copy it in to python or test it on a virtual machine like I did.
If you are still concerned... don't download it but if you want to or are still anxious test it on a virtual machine.

**Comment by MC-HackerTech** [comment]
Which exact file is a backdoor?
I will check the code

> Note : REMEMBER EVEN IF THE CODE IS OPEN SOURCE IT COULD BE A MALWARE

**Comment by MC-HackerTech** [comment]
> This program is a HTTP/HTTPS backdoor into your computer! Dont download!

Hmm I don't think there is a .exe file anywhere in the repositor. Which one exactly the backdoor file/script ?

**Comment by Elara6331** [comment]
> > This program is a HTTP/HTTPS backdoor into your computer! Dont download!
> 
> Hmm I don't think there is a .exe file anywhere in the repository. Which one exactly the backdoor file/script ?

There is no executable (Windows or otherwise) in the repo. I don't believe the issue creator actually knows which one is a backdoor, and I think they believe that the repo itself is a backdoor (as in cloning the repo installs a backdoor), which is why I tried to explain that a backdoor cannot be installed by simply cloning a repo.

**Comment by dagod1** [comment]
can i ask if people can track this if you dont have a vpn while on the files

**Comment by ghost** [comment]
lol

**Comment by worstrojan** [comment]
> > > This program is a HTTP/HTTPS backdoor into your computer! Dont download!
> > 
> > Hmm I don't think there is a .exe file anywhere in the repository. Which one exactly the backdoor file/script ?
> 
> There is no executable (Windows or otherwise) in the repo. I don't believe the issue creator actually knows which one is a backdoor, and I think they believe that the repo itself is a backdoor (as in cloning the repo installs a backdoor), which is why I tried to explain that a backdoor cannot be installed by simply cloning a repo.

you have a fucking brain my guy 🙂

**Comment by Reverrb** [comment]
the hulk one seems legit, since he explains what he's doing in certain parts of the script. I'm gonna go try it and see for myself

**Comment by Reverrb** [comment]
well update, tried 6 on here and only works against http and not https, also its only a dos. Conclusion, Don't download

**Comment by ghost** [comment]
OK, THIS IS SAFE! check task manager for any hidden processes if you don't belive me, or wireshark.
